plugins/scaffolder-backend/src/scaffolder/stages/publish/github.ts

```diff
@@ -44,13 +44,14 @@ export class GithubPublisher implements PublisherBase {
     values: RequiredTemplateValues & Record<string, JsonValue>,
   ) {
     const [owner, name] = values.storePath.split('/');
+    const description = String(values.description);
```

Comment:
I don't really understand this part of Typescript. description has type `JsonValue` so I have to convert it into a string or the `createInOrg` method won't accept it. It expects a string. But surely there is a nicer way to handle this?

Reply:
I think the issue is this. The values object is parsed from json so the signature here gives explicit types to the two required params and just says there maybe other stuff in the json object (Record<string, JsonValue>). Since the other keys are unknown and could be anything (objects, arrays, booleans  or strings) that's why you've got JsonValue as the type. I think the thing to do would be have an OptionalTemplateValues type with { description?: string } but createInOrgs signature might not like passing in undefined (rather than string - haven't looked that far into it)

Reply:
`createInOrgs` will accept `undefined`. VSCode told me so! 😄 

I still don't really get it. I'm explicitly only passing the `description` into `createInOrg` so it shouldn't care what the rest of the `values` object contains. 🤷  I'll ship it over to spotify/backstage and see what the experts there say.